docs/how-to/upgrade.md

```diff
@@ -1,4 +1,8 @@
-# Migrate from COS Lite to COS
+# Upgrade instructions
+
+## COS 2
+
+### Migrate from COS Lite 1 to COS 2
```

Comment:
`COS 2`?
`COS Lite 1`?

What do we mean, `COS` deployment using track/2, and `COS Lite` deployment using track/1 ?